agents/ls-json/src/main/resources/org.eclipse.che.ls.json.json

```diff
@@ -1,5 +1,6 @@
 {
   "id": "org.eclipse.che.ls.json",
+  "version": "1.0.1",
```

Comment:
Can you explain why it should have `1.0.1` but not `1.0.0` while `org.eclipse.che.ls.csharp.json` has `1.0.0`